### PR TITLE
Updated log message to inform about incorrect CL for read repair (3.x)

### PR DESCRIPTION
**Pull request by AnkitBarsainyaPSL**
What was the issue?
During read repair failure, incorrect CL is being generated which needs correction.

What was fixed?
During read repair failure, incorrect CL is being generated. To rectify this we're adding additional clarification to not refer to CL logged in the error message in case of Read Repair Failure.

### CODE REVIEW

**Review by adutra — APPROVED** [review]

File: driver-core/src/main/java/com/datastax/driver/core/exceptions/ReadTimeoutException.java [hunk 1]
```diff
@@ -42,8 +42,11 @@ public ReadTimeoutException(
     super(
         endPoint,
         String.format(
+            "Cassandra timeout during read query at consistency %s (%s). "
+                + "In case this was generated during read repair, the consistency level is not representative of the actual consistency.",
             "Cassandra timeout during read query at consistency %s (%s)",
```
Comment (justinchuch) [inline driver-core/src/main/java/com/datastax/driver/core/exceptions/ReadTimeoutException.java]: this is a redundant line. I believe this line is supposed to be replaced by the new changes.

**Review by justinchuch — CHANGES_REQUESTED** [review]

File: driver-core/src/main/java/com/datastax/driver/core/exceptions/ReadTimeoutException.java [hunk 2]
```diff
@@ -42,8 +42,11 @@ public ReadTimeoutException(
...
-            consistency, formatDetails(received, required, dataPresent)),
+            consistency,
```
Comment (berndocklin) [inline driver-core/src/main/java/com/datastax/driver/core/exceptions/ReadTimeoutException.java]: another comment is that this is an unnecessary formatting change not related to the problem we want to fix. please remove.

File: driver-core/src/main/java/com/datastax/driver/core/exceptions/ReadTimeoutException.java (hunk 1 above)
Comment (AnkitBarsainyaPSL) [reply driver-core/src/main/java/com/datastax/driver/core/exceptions/ReadTimeoutException.java]: Thanks for pointing out. Probably skipped while switching/raising multiple PR on this. Taken care now.

File: driver-core/src/main/java/com/datastax/driver/core/exceptions/ReadTimeoutException.java (hunk 2 above)
Comment (AnkitBarsainyaPSL) [reply driver-core/src/main/java/com/datastax/driver/core/exceptions/ReadTimeoutException.java]: Thanks. Taken care of now.

**Review by adutra — APPROVED** [review]

**Review by justinchuch — APPROVED** [review]

**Comment by justinchuch** [comment]
maybe squash the commits into one after approval?
